mapper_windows_test.go

```diff
@@ -33,11 +33,12 @@ func TestWindowsFileMapper(t *testing.T) {
 	p := mustNew(t, &cli)
 	_, err := p.Parse([]string{"testdata\\file.txt"})
 	assert.NoError(t, err)
-	assert.NotNil(t, cli.File)
+	assert.True(t, cli.File != nil, "File should not be nil")
```

Comment:
This is fine, but you should be able to use `assert.NotZero()` here if you prefer.

Reply:
🤦  Doh. I completely blanked on that. "Hm, there's no NotNil. Guess that's not a covered use case."

Reply:
Fixed here https://github.com/alecthomas/kong/pull/396